microsoft-365/enterprise/manage-multitenant-org-settings.md

```diff
@@ -159,8 +159,21 @@ These permissions are required to fetch cross-tenant synchronization details and
   
   - [Synchronization.Read.All](/graph/permissions-reference#synchronizationreadall)
   
+
 This permission is required to gather details regarding the multitenant organization.
 
 - Reading MTO details 
 
   - [MultiTenantOrganization.Read.All](/graph/permissions-reference#multitenantorganizationreadall)
+
+## Manage external tag removal (Private Preview)
```

Comment:
I would reword this to say "Manage Outlook external tag removal for MTO members (Private Preview)" to make it clear that this applies to Outlook specifically.